Assets/SibylSystem/Program.cs

```diff
@@ -429,6 +1531,86 @@ private void UpdateClient()
             }
         }
     }
+    private void UpdateClientV2()
+    {
+        if (UIHelper.fromStringToBool(Config.Get("autoUpdateDownload_", "1")))
+        {
+            try
+            {
+                WWW w = new WWW("https://api.github.com/repos/szefo09/updateYGOPro2/contents/");
+                while (!w.isDone)
+                {
+                    if (Application.internetReachability == NetworkReachability.NotReachable || !string.IsNullOrEmpty(w.error))
+                    {
+                        throw new Exception("No Internet connection!");
+                    }
+                }
+                List<ApiFile> toDownload = new List<ApiFile>();
+                List<ApiFile> apiFromGit = new System.Web.Script.Serialization.JavaScriptSerializer().Deserialize<List<ApiFile>>(w.text);
+
+                List<string> local = new List<string>();
+                local.AddRange(new DirectoryInfo("config/").GetFiles("*.conf").Select(x => x.Name).ToList());
+                local.AddRange(new DirectoryInfo("cdb/").GetFiles("*.cdb").Select(x => x.Name).ToList());
+                foreach (ApiFile file in apiFromGit)
+                {
+                    string s = local.FirstOrDefault(x => x == file.name);
+                    if (s == null)
+                    {
+                        toDownload.Add(file);
+                    }
+                    else
+                    {
+                        FileInfo f = new FileInfo((Path.GetExtension(s).ToLower() == ".cdb" ? "cdb/" : "config/") + s);
+                        byte[] bytes1 = System.Text.Encoding.ASCII.GetBytes("blob " + f.Length.ToString() + '\0'.ToString());
+                        byte[] bytes2 = File.ReadAllBytes((Path.GetExtension(s).ToLower() == ".cdb" ? "cdb/" : "config/") + s);
+                        List<byte> temp = new List<byte>();
+                        temp.AddRange(bytes1);
+                        temp.AddRange(bytes2);
+                        byte[] bytes = temp.ToArray();
+                        string sha = GetHashString(GetHash(bytes)).ToLower(); ;
+                        if (sha != file.sha)
+                        {
+                            toDownload.Add(file);
+                        }
+                    }
+                }
+                //foreach (string f in local)
+                //{
+                //    if (apiFromGit.FirstOrDefault(x => x.name == f) == null || f != apiFromGit.FirstOrDefault(x => x.name == f).name)
```

Comment:
Why do you no longer want to remove unnecessary files?

Reply:
Because it deletes config.conf because it isnt on the udpate git